privly.safariextension/scripts/background_scripts/show_popover.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @fileOverview This file shows the popover when the toolbar item is
+ * clicked. Command events are generated when the user clicks an
+ * extension's toolbar item.
+ * More about command events can be read about at
+ * https://developer.apple.com/library/safari/documentation/Tools/Conceptual/SafariExtensionGuide/SafariExtensionGuide.pdf
+ */
+
+/**
+ * Shows the popover based on the command event.
+ *
+ * @param event the command event that is dispatched
+ *
+ */
+function performCommand(event) {
```

Comment:
performCommand is barely more descriptive than runFunction. You need a better/more descriptive name here, like `showPopoverCallback()`.